plugins/callback/splunk.py

```diff
@@ -116,7 +127,11 @@ def send_event(self, url, authtoken, validate_certs, state, result, runtime):
         data['uuid'] = result._task._uuid
         data['session'] = self.session
         data['status'] = state
-        data['timestamp'] = datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S '
+        if include_milliseconds:
+            data['timestamp'] = datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S.%f '
```

Comment:
The only difference between the if and else block is the format string. Maybe set only that format string in the if/else block and call strftime afterwards?